src/components/Layout.js

```diff
@@ -1,12 +1,10 @@
-import Footer from './Footer';
 import Header from './Header';
 
 export default function Layout({children}) {
 	return (
 		<>
-			<Header />
 			<main>{children}</main>
-			<Footer />
+			<Header />
```

Comment:
Do you really want to render your `Header` component below the `main` element? 🤔